server.js

```diff
@@ -1,30 +1,129 @@
 import express from 'express'
-import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
+import listEndpoints from 'express-list-endpoints'
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true, useCreateIndex: true})
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
 const port = process.env.PORT || 8080
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
+//schema
+const thoughtSchema = new mongoose.Schema({
+  message: {
+    type: String,
+    required: [true, 'did someone forget to type in a message? (someone = you)'],
+    unique: true,
```

Comment:
The message is not a good field to make it unique. if 3 recruiters come and send "testing" 2 of them will think it is not working 😭